src/komponenter/Kalkulator/Kalkulator.tsx

```diff
@@ -29,6 +30,11 @@ export const Fraværskalulator: FunctionComponent<
 
   return (
     <div className={styles.wrapper}>
+      <TestVersjonBanner
+        sidenavn="Fraværskalkulator"
```

Comment:
Syntes det virker litt stakkato med 

> Her kan du bli bedre kjent med siden Fraværskalkulator

Jeg ville kanskje endret til 

> Her kan du bli bedre kjent med fraværskalkulatoren

NAV har forøvrig en policy om å skrive produktene/tjenestene sine med liten forbokstav når det er midt i en setning

Reply:
Takk, den policyen var jeg ikke klar over. Jeg har lagt til en endring for det.